test/system/037-runlabel.bats

```diff
@@ -12,7 +12,7 @@ load helpers
     rand3=$(random_string 30)
     cat >$containerfile <<EOF
 FROM $IMAGE
-LABEL  INSTALL  /usr/bin/podman  run  -t  -i  --rm  \\\${OPT1}  --privileged  -v /:/host --net=host --ipc=host --pid=host -e HOST=/host -e NAME=\\\${NAME} -e IMAGE=\\\${IMAGE} -e CONFDIR=/etc/\\\${NAME} -e LOGDIR=/var/log/\\\${NAME} -e DATADIR=/var/lib/\\\${NAME} \\\${IMAGE} \\\${OPT2} /bin/install.sh \\\${OPT3}
+LABEL  INSTALL  $PODMAN  run  -t  -i  --rm  \\\${OPT1}  --privileged  -v /:/host --net=host --ipc=host --pid=host -e HOST=/host -e NAME=\\\${NAME} -e IMAGE=\\\${IMAGE} -e CONFDIR=/etc/\\\${NAME} -e LOGDIR=/var/log/\\\${NAME} -e DATADIR=/var/lib/\\\${NAME} \\\${IMAGE} \\\${OPT2} /bin/install.sh \\\${OPT3}
```

Comment:
It is not well documented but this should just use `podman`, while your change will work in the test podman is already smart enough to replace `podman` with `/proc/self/exe` when it is the first arg.
https://github.com/containers/podman/blob/9062dede656933646079e4eabeecf3b8fc77ca63/pkg/domain/infra/abi/containers_runlabel.go#L263-L270

Reply:
Thanks. Fixed to just `podman` and repushed.